docs/sources/insights-and-metrics/_index.md

```diff
@@ -177,7 +177,8 @@ Logs contain the following fields, where the fields followed by * are always ava
 
 resource types are: `integration_heartbeat`, `escalation_chain`, `integration`, `outgoing_webhook`,
 `escalation_policy`, `public_api_token`, `schedule_export_token`,`user_schedule_export_token`,
-`oncall_shift`, `web_schedule`, `ical_schedule`, `calendar_schedule`, `organization`, `user`, `webhook`.
+`oncall_shift`, `web_schedule`, `ical_schedule`, `calendar_schedule`, `shift_swap_request`, `organization`,
```

Comment:
should shift_swap_request be shift_swap as that covers the features more fully?
(i know this would be a code change, not just a doc change, but i'll just mention it here and hope you can take it from here)

Reply:
This is the resource name in the insight logs, which (usually) match the model name (as you can guess from the other resource names), so I would prefer to keep it in this case. Also, the log is for the request itself, so I think it makes sense to keep it like that? (I mean, these log entries track the data and status of a request)